nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -139,9 +141,7 @@ def broadcast_model(
         if not blocking and not isinstance(callback, Callable):
             raise TypeError("callback must be defined if blocking is False, but got {}".format(type(callback)))
 
-        if not data:
-            self.warning("data is None. Sending empty FLModel.")
-            data = FLModel(params_type=ParamsType.FULL, params={})
```

Comment:
We probably need to keep this code.

Reply:
Since we check the "not data" to raise exception, we don't need this code any more.